services/QuillLMS/client/package.json

```diff
@@ -6,8 +6,10 @@
     "url": "https://github.com/empirical-org/Empirical-Core.git"
   },
   "license": "AGPL-3.0",
+  "engineStrict" : true,
```

Comment:
Setting this flag to `true` will cause npm to throw exceptions when `engines` directives are not met. This line isn't necessary, but I found it gave me peace of mind while making this config change to know that my intended version of npm was being used. Feel free to chime in about keeping this flag or not.

Reply:
This seems reasonable to me, assuming that it doesn't cause issues during the build process.